db/migrate/20181115194822_add_borrow_direct_boolean_column_to_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddBorrowDirectBooleanColumnToRequests < ActiveRecord::Migration
+  def change
+    add_column :requests, :via_borrow_direct, :boolean, default: false
```

Comment:
I've added this migration in a separate commit, and am totally willing to remove it.

At first, I was thinking that there would be state where we might want to set the via_borrow_direct flag at some point before making the official request to them, but it's all pretty much at the same time as far as the user will be concerned, so we could theoretically infer this from the existence of any `borrow_direct_response_data`.